docs/src/installation.rst

```diff
@@ -30,6 +30,8 @@ Installing the controller
 
 You can install the development version of the CARTA controller from NPM by running ``npm install -g carta-controller@dev``, or from GitHub by cloning the `controller repository <https://github.com/CARTAvis/carta-controller/>`_ and running ``npm install``.
 
+Specific authentication methods may have additional system dependencies. For example, the default PAM authentication method requires ``libpam0g-dev`` on Ubuntu.
```

Comment:
since we're going for pam as a good starting point, @robsimmonds suggested we keep it as a fixed (rather than optional) dependency for now

Reply:
I think that will only really become relevant when we create the apt package for the controller (at which point we can decide on the exact dependency chain). Here the user is still manually installing apt dependencies. I have included this in the list of packages to install in the detailed Focal instructions; this is general information which does not assume a particular auth method (e.g. for direct LDAP they wouldn't need the PAM package but might need additional LDAP utilities).

Reply:
Is the problem that you're making an NPM PAM package a fixed dependency of the controller, and that will fail if the apt dependency is not installed, even if a different auth method is used?

Reply:
I guess this can be resolved now :)